### PR TITLE
GATK ASEReadCounter recipe, resolves #503

### LINKED ISSUE #503: Add ASEReadCounter recipe

### PR DESCRIPTION
**Pull request by jemten**

### CODE REVIEW

**Comment by henrikstranneheim** [comment]
Let me know when this one is ready

**Comment by jemten** [comment]
Ready for review!

File: lib/MIP/Program/Variantcalling/Salmon.pm [hunk 1]
```diff
@@ -203,12 +203,12 @@ sub salmon_quant {
...
     push @commands, q{--index} . $SPACE . $index_path;
 
-    # Library type, defines if the library is stranded or not, and the orientation of the reads, according to the documentation http://salmon.readthedocs.io/en/latest/library_type.html
+# Library type, defines if the library is stranded or not, and the orientation of the reads, according to the documentation http://salmon.readthedocs.io/en/latest/library_type.html
     push @commands, q{--libType} . $SPACE . $lib_type;
 
     push @commands, q{--output} . $SPACE . $outfile_path;
 
-    # The input Fastq files, either single reads or paired. Salmon uses a bash command to stream the reads. Here, the default is <( bunzip2 file.fastq.gz )
+# The input Fastq files, either single reads or paired. Salmon uses a bash command to stream the reads. Here, the default is <( bunzip2 file.fastq.gz )
```
Comment (henrikstranneheim) [inline lib/MIP/Program/Variantcalling/Salmon.pm]: The default seem to have changed!

File: lib/MIP/Recipes/Analysis/Gatk_asereadcounter.pm [hunk 2]
```diff
@@ -0,0 +1,384 @@
...
+                $sitesfile_prefix
+                  . substr( $sitesfile_suffix, 0, 2 )
+                  . $ASTERIX
+            ),
+            outfile_path => $temp_directory,
+        }
+    );
+    say {$FILEHANDLE} q{wait} . $NEWLINE;
+
+    ## GATK HaplotypeCaller
```
Comment (henrikstranneheim) [inline lib/MIP/Recipes/Analysis/Gatk_asereadcounter.pm]: HaplotypeCaller -> ASEReadCounter

**Comment by henrikstranneheim** [comment]
👍

**Pull request merged**